addons/point_of_sale/models/pos_payment_method.py

```diff
@@ -75,7 +77,16 @@ def _load_pos_data_domain(self, data, config):
 
     @api.model
     def _load_pos_data_fields(self, config):
-        return ['id', 'name', 'is_cash_count', 'use_payment_terminal', 'split_transactions', 'type', 'image', 'sequence', 'payment_method_type', 'default_qr']
+        return ['id', 'name', 'is_cash_count', 'use_payment_terminal', 'split_transactions', 'type', 'image', 'sequence', 'payment_method_type', 'default_qr', 'qr_code_method_icon_ids']
```

Comment:
```suggestion
        return ['id', 'name', 'is_cash_count', 'use_payment_terminal', 'split_transactions', 'type', 'image', 'sequence', 'payment_method_type', 'default_qr']
```

it will fetch base64 data for all icons so no need to load it